.github/workflows/bump-version-release.yaml

```diff
@@ -44,10 +44,13 @@ on:
         description: "The old version in your `.bumpversion.cfg` file."
         value: ${{ jobs.release.outputs.old-version }}
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-bump-version-release
+  cancel-in-progress: true
+
```

Comment:
We do not want workflows that create tags or releases to every be canceled while they are running